docs/translations.md

```diff
@@ -0,0 +1,54 @@
+# How to contribute translations
+
+There are several things you can do to help us internationalize Brave and provide a great experience for everybody
+
+## Providing a translation
```

Comment:
We generally pull in all languages files at the time we cut a release. That allows us to keep everything up to date in a scalable way.